assets/src/edit-story/elements/text/updateForResizeEvent.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { MIN_FONT_SIZE, MAX_FONT_SIZE } from '../../constants';
+import {
+  calculateFitTextFontSize,
+  calculateTextHeight,
+} from '../../utils/textMeasurements';
+import { dataPixels } from '../../units/dimensions';
+
+function updateForResizeEvent(element, newWidth, newHeight) {
+  const { width, height } = element;
+  const isResizingWidth = Math.abs(newWidth - width) >= 1;
```

Comment:
Wouldn't it be safer to determine this by directions? In theory, it's possible to resize only one side, e.g. only width when resizing from corners, which would then act as if resizing would happen from one edge only.

Reply:
Yeah. I think you're right. I'll redo this piece by passing the directions array.

Reply:
I'm passing the directions explicitly now. Definitely a lot safer and better. Thanks! https://github.com/google/web-stories-wp/pull/406